pages/manual/manual-gettingstarted-build.md

```diff
@@ -100,22 +110,56 @@ Unless you specifically need to debug the application, we recommend setting the
 
 
 ### Mac
-TODO: do
-{% include note.html content="
-When using a multi-configuration generator (like Xcode) you may want to adjust your build mode manually, since it probably defaults to `Debug` at first, which has a large impact on the performance.
-
-If you use a single-configuration generator, you can control the build mode using `CMAKE_BUILD_TYPE` in CMake.
+#### Dependencies
+You will need at least
+- [XCode](https://developer.apple.com/xcode/) 
+- [CMake](https://cmake.org/download/) >= 3.12.0
```

Comment:
I really dislike that we have this stated here. Since we never test that that is true. I would be more honest to write "a recent version" of "one of the latests versions". Anything else is just pointless, since we have no idea if it is true or not.

Reply:
same goes for the win dependences also

Reply:
Jenkins should run with the CMake version we specify here in my opinion.

Reply:
Jenkins uses 3.22. I needs to stay quite recent to keep up with Xcode updates etc. Which needs to keep up with OS updates. Which we have to keep updated since liu-it, since security...

Reply:
Ok. I have added that we recommend using the latest version for all dependencies

Reply:
Actually I don't think 3.12 will work. you probably need at least 3.15.